lib/templates/structure/widget-area.php

```diff
@@ -3,93 +3,102 @@
+echo beans_get_widget_area( 'before_widgets' ); // @codingStandardsIgnoreLine - WordPress.XSS.EscapeOutput.OutputNotEscaped.
+
+if ( 'grid' === beans_get_widget_area( 'beans_type' ) ) {
+	beans_open_markup_e(
+		'beans_widget_area_grid' . _beans_widget_area_subfilters(),
+		'div',
+		array(
+			'class'               => 'uk-grid',
+			'data-uk-grid-margin' => '',
+		)
+	);
+}
+
+if ( 'offcanvas' === beans_get_widget_area( 'beans_type' ) ) {
+
+	beans_open_markup_e(
+		'beans_widget_area_offcanvas_wrap' . _beans_widget_area_subfilters(),
+		'div',
+		array(
 			'id'    => beans_get_widget_area( 'id' ), // Automatically escaped.
 			'class' => 'uk-offcanvas',
-		) );
-
-			beans_open_markup_e( 'beans_widget_area_offcanvas_bar' . _beans_widget_area_subfilters(), 'div', array( 'class' => 'uk-offcanvas-bar' ) );
+		)
+	);
 
-	}
+		beans_open_markup_e( 'beans_widget_area_offcanvas_bar' . _beans_widget_area_subfilters(), 'div', array( 'class' => 'uk-offcanvas-bar' ) );
+}
 
 		// Widgets.
-		if ( beans_have_widgets() ) :
+if ( beans_have_widgets() ) :
 
-			/**
+	/**
 			 * Fires before widgets loop.
 			 *
 			 * This hook only fires if widgets exist.
 			 *
 			 * @since 1.0.0
 			 */
-			do_action( 'beans_before_widgets_loop' );
+	do_action( 'beans_before_widgets_loop' );
 
-				while ( beans_have_widgets() ) : beans_setup_widget();
+	while ( beans_have_widgets() ) :
+		beans_setup_widget();
 
-					if ( 'grid' == beans_get_widget_area( 'beans_type' ) ) {
-						beans_open_markup_e( 'beans_widget_grid' . _beans_widget_subfilters(), 'div', beans_widget_shortcodes( 'class=uk-width-medium-1-{count}' ) );
-					}
+		if ( 'grid' === beans_get_widget_area( 'beans_type' ) ) {
+			beans_open_markup_e( 'beans_widget_grid' . _beans_widget_subfilters(), 'div', beans_widget_shortcodes( 'class=uk-width-medium-1-{count}' ) );
+		}
 
-						beans_open_markup_e( 'beans_widget_panel' . _beans_widget_subfilters(), 'div', beans_widget_shortcodes( 'class=tm-widget uk-panel widget_{type} {id}' ) );
+			beans_open_markup_e( 'beans_widget_panel' . _beans_widget_subfilters(), 'div', beans_widget_shortcodes( 'class=tm-widget uk-panel widget_{type} {id}' ) );
 
-							/**
+				/**
 							 * Fires in each widget panel structural HTML.
 							 *
 							 * @since 1.0.0
 							 */
-							do_action( 'beans_widget' );
+				do_action( 'beans_widget' );
 
-						beans_close_markup_e( 'beans_widget_panel' . _beans_widget_subfilters(), 'div' );
+			beans_close_markup_e( 'beans_widget_panel' . _beans_widget_subfilters(), 'div' );
 
-					if ( 'grid' == beans_get_widget_area( 'beans_type' ) ) {
-						beans_close_markup_e( 'beans_widget_grid' . _beans_widget_subfilters(), 'div' );
-					}
+		if ( 'grid' === beans_get_widget_area( 'beans_type' ) ) {
+			beans_close_markup_e( 'beans_widget_grid' . _beans_widget_subfilters(), 'div' );
+		}
+endwhile;
 
-				endwhile;
-
-			/**
+	/**
```

Comment:
The rest of this comment needs to align left too to align with its opening `/**`.